WordPress/Classes/ViewRelated/Jetpack/Jetpack Scan/View Models/JetpackScanThreatViewModel.swift

```diff
@@ -309,8 +311,128 @@ struct JetpackScanThreatViewModel {
     }
 }
 
+extension JetpackScanThreatViewModel {
+
+    private static func attributedString(for context: JetpackThreatContext?) -> NSAttributedString? {
+        guard let context = context else {
+            return nil
+        }
+
+        let lineFont = UIFont.monospacedSystemFont(ofSize: 13, weight: .semibold)
+        let numberFont = UIFont.monospacedSystemFont(ofSize: 13, weight: .bold)
+        let attrString = NSMutableAttributedString()
+
+        let biggestLen = String(context.lines.last?.lineNumber ?? 0).count
+
+        let longestLine = context.lines.sorted { $0.contents.count > $1.contents.count}.first!
```

Comment:
This line causes a crash if the context.lines is an empty array. We could probably guard or default instead of using a forced unwrap.